.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+Metrics/LineLength:
```

Comment:
This file can probably go away with cookstyle. We set reasonable defaults

Reply:
Ok ;)